freqtrade/exchange/exchange.py

```diff
@@ -539,7 +539,9 @@ def get_min_pair_stake_amount(self, pair: str, price: float,
         # reserve some percent defined in config (5% default) + stoploss
         amount_reserve_percent = 1.0 + self._config.get('amount_reserve_percent',
                                                         DEFAULT_AMOUNT_RESERVE_PERCENT)
-        amount_reserve_percent += abs(stoploss)
+        amount_reserve_percent = (
+          amount_reserve_percent / (1 - abs(stoploss)) if abs(stoploss) != 1 else 1.5
+        )     
```

Comment:
Apparently this line has a trailing whitespace (and one of the test lines is > 100 characters) - which makes CI fail.

Apart from that, i do think you're right (and the calculation confirms this as well) - so thanks for looking into this!

Reply:
Yep, fixing that now. Thanks for your patience and for the library :) 
BTW, in your opinion, is a PR the best practice to fix those kind of small quirks ?

Reply:
define "small quirks" ? you mean the CI errors - or the calculation problem?

a PR is usually the best way to make changes to a project. obviously, an issue is also possible if you discover a problem but are unsure about a fix - but if it's a one-line fix (like in this case) - a PR is the quickest way to point out the problem, while at the same  time fixing the problem.

Now ideally, ci passes right away - but i often forget to run the tools we use myself - however that's what we got CI for - to tell us that there's some small stylistic errors (without much discussion, as CI tells us what's wrong).